lib/filters/organization_filter.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveInteraction
+  class OrganizationFilter < Filter
+    register :organization
+
+    def cast(value, context)
+      schema = OrganizationUpdateSchema.schema[:properties]
+      included_keys = value.keys.map(&:to_sym)
+      if included_keys.all? {|s| schema.key? s } && valid?(value)
+        value
+      else
+        raise InvalidValueError, "#{name}: #{describe(value)}"
+      end
+    end
+
+    def valid?(value)
+      JSON::Validator.validate(OrganizationUpdateSchema.schema, value)
```

Comment:
This does something different than the controller counterpart at https://github.com/zooniverse/Panoptes/blob/master/lib/json_api_controller/json_schema_validator.rb#L43.

That `validate!` method is defined here: https://github.com/zooniverse/Panoptes/blob/master/lib/json_schema.rb#L89, and again, that's different than what's here. I'm not sure what the difference between `validate` and `fully_validate` is. Also, the implementation of that `validate!` suggests that `.fully_validate` might not return true/false depending on validation status?

Tests for this would make things clear (and keep stuff working if we upgrade the validator gem.